pkgs/tools/inputmethods/uim/default.nix

```diff
@@ -6,7 +6,7 @@
 , withGtk ? true
 , withGtk2 ? withGtk, gtk2 ? null
 , withGtk3 ? withGtk, gtk3 ? null
-, withQt ? true
+, withQt ? false # qt5 is broken and qt4 removed -> No qt support possible
```

Comment:
Seriously? `qt5.qtbase` is broken?

Reply:
qt5 support in uim is broken in Nixpkgs.

Reply:
TBH the comment you phrased seems more confusing then explaining. First of all, there is no reference to qt4 in the expression. The `withQt` seems to me like an old argument that was replaced / superseded at some point by `withQt5`. Hence, I'd remove completely `withQt` in favor of `withQt5`. Hence I'd rephrase the comment as follows:

```
# qt5 support is broken in uim
```

No need to mention qt4, and the word support is important here.

Reply:
qt5 support is not broken if it's just missing the wrapper from nativeBuildInputs because it can be fixed by adding it.